### ISSUE TITLE
Allow the situation when Condition Era is not built from Condition occurrence where condition_concept_id =0

### ISSUE THREAD

**Issue by dimshitc** [OHDSI/DataQualityDashboard#493]
Please edit the measureConditionEraCompleteness checl

The number and Percent of persons that does not have condition_era built successfully for all persons in condition_occurrence.
It should omit Condition records where condition_concept_id = 0. In this case it's OK, that era is not built

**Comment by katy-sadowski** [comment]
Agree, good catch. People whose only CO rows have 0 concept_id, and people who have no conditions at all (already excluded from the check), should not be expected to have condition eras. And we'll flag the missed standard concept mapping in another check.

**Comment by MaximMoinat** [comment]
Note that the default condition era derivation script does NOT filter for condition_concept_id = 0:
https://ohdsi.github.io/CommonDataModel/sqlScripts.html#Condition_Eras
